README.md

```diff
@@ -23,6 +23,17 @@ It was created as an answer to deeply recursive Pyanodon recipes, which the tool
 
 Among other things YAFC has Never Enough Items, which is FNEI on steroids. In addition to showing the recipes, it shows which ones you probably want to use, and how much.
 
+## **[Download YAFC and Getting Started](https://github.com/have-fun-was-taken/yafc-ce/releases)**
+
+YAFC is a desktop app. Windows build is the most tested, but OSX and Linux are there too. See [Linux and OSX installation instructions](/Docs/LinuxOsxInstall.md).
+
+1. Go to the [releases page](https://github.com/have-fun-was-taken/yafc-ce/releases)
+2. Download the zip file for your given OS
+3. Extract the zip file to your preferred location on your computer
```

Comment:
```suggestion
2. Download the zip file for your given OS,
3. Extract the zip file to your preferred location,
```
This list is punctuated by commas at the end for the sentences to not be left hanging.